sdk/python/lib/pulumi/runtime/mocks.py

```diff
@@ -31,9 +32,12 @@
 
 
 def test(fn):
+
+    @functools.wraps(fn)
```

Comment:
The effect of this decorator is such that the wrapped function preserves the same signature (such as argument names) as the original. This ends up being important for name-based fixture matching functionality of pytest. So now when I write

```
@pulumi.runtime.test
def test_foo(my_fixture):
   ...
```

It works, which it didn't previously.